nncf/onnx/quantization/quantize_model.py

```diff
@@ -75,3 +82,91 @@ def quantize_impl(
     quantized_model = quantization_algorithm.apply(model, graph, dataset=calibration_dataset)
 
     return quantized_model
+
+
+def quantize_with_accuracy_control_impl(
+    model: onnx.ModelProto,
+    calibration_dataset: Dataset,
+    validation_dataset: Dataset,
+    validation_fn: Callable[[Any, Iterable[Any]], Tuple[float, Union[None, List[float], List[List[TTensor]]]]],
+    max_drop: float = 0.01,
+    drop_type: DropType = DropType.ABSOLUTE,
+    preset: Optional[QuantizationPreset] = None,
+    target_device: TargetDevice = TargetDevice.ANY,
```

Comment:
Can we duplicate warning messages and excpetion raisers from `quantize_impl(...)`?
I would put them in the single function